packages/application-config/src/process-config.ts

```diff
@@ -104,8 +117,14 @@ const processConfig = ({
   // This is important to determine to which URL the MC should redirect to
   // after successful login.
   const applicationId = isProd
-    ? `${customApplicationData.id}:${customApplicationData.entryPointUriPath}`
-    : `__local:${customApplicationData.entryPointUriPath}`;
+    ? `${configurationData.id}:${
+        (configurationData as CustomApplicationData).entryPointUriPath ||
+        'custom-view'
+      }`
+    : `__local:${
+        (configurationData as CustomApplicationData).entryPointUriPath ||
+        configurationData.id
```

Comment:
For Custom Views, we use their ID since they don't have an entry point.

Reply:
We would be creating a fake ID if it does not exist in the config file.
Currently, it will be just a timestamp.

Please take a look at line 25 in the template [entry point file](https://github.com/commercetools/merchant-center-application-kit/pull/3175/files/42d36ac7b1891f6366cfc6ff946ab1f389b2619a#diff-b5adf3d372fa6efa1c98b999fea58ccd2de7b26112c36f095e89f495ace3df8d).